### PR TITLE
chore(ci): ignore perception/bytetrack/lib in spell-check-partial

### PR DESCRIPTION
**Pull request by HansRobo**
## Description

Ignore cspell errors in `perception/bytetrack/lib`, because the files are from external source and we want to keep the spell errors as a kind of bug-compatibility. 

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: .cspell-partial.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "ignorePaths": ["**/perception/**", "**/planning/**", "**/sensing/**"],
-  "ignoreRegExpList": [],
+  "ignoreRegExpList": ["perception/bytetrack/lib/**"],
```
Comment (kminoda) [inline .cspell-partial.json:3]: May I ask why here instead of "ignorePaths"?

**Review by kminoda — COMMENTED** [review]
Thank you!! Let me also add @manato as a reviewer, as he is the maintainer of the package.

File: .cspell-partial.json (hunk 1 above)
Comment (HansRobo) [reply .cspell-partial.json:3]: Oh, it's my mistake!
Thank you

Comment (HansRobo) [inline .cspell-partial.json:3]:
```suggestion
  "ignoreRegExpList": [],
```

Comment (HansRobo) [reply .cspell-partial.json]: FIxed in https://github.com/autowarefoundation/autoware.universe/pull/4517/commits/86c558060b0e9e4a49f4f99f298dd1753d5180c1

**Review by kminoda — COMMENTED** [review]
Sorry for being late. LGTM!

But let me confirm someone who's in charge of this package: @manato @miursh @Shin-kyoto 
Could you review this PR for us?

--
I think there were some options to address this bytetrack spell-check issues.

1. Ignore some part of bytetrack (like in this PR)
2. Add a dedicated dictionary for this package
3. Fix all the spell-checks (discarding the upstream compatibility)

Would be great if you could share some rationale behind the decision among these as well :pray: (Happy to consult with these options in detail in person if necessary)

**Review by miursh — APPROVED** [review]
Ignoring ported library of bytetrack is good idea.
I think T4 perception team has discussed and agrees with this option since the codes in bytetrack/lib/ are all copied from https://github.com/ifzhang/ByteTrack.
Fixing spells in it is low priority because it is not essential.

**Pull request merged**